pepper-apis/src/main/java/org/broadinstitute/ddp/Housekeeping.java

```diff
@@ -665,6 +681,21 @@ public void receiveMessage(PubsubMessage message, AckReplyConsumer consumer) {
         }
     }
 
+    private static void createActivityInstance(Handle apisHandle, EventConfigurationDto eventConfDto, QueuedEventDto pendingEvent) {
+        User participant = apisHandle.attach(UserDao.class)
+                .findUserByGuid(pendingEvent.getParticipantGuid())
+                .orElse(null);
+        StudyDto studyDto = apisHandle.attach(JdbiUmbrellaStudy.class).findByStudyGuid(pendingEvent.getStudyGuid());
```

Comment:
Might as well use the cached version of this:
```suggestion
        StudyDto studyDto = new JdbiUmbrellaStudyCached(apisHandle).findByStudyGuid(pendingEvent.getStudyGuid());
```